contrib/depends/packages/packages.mk

```diff
@@ -1,15 +1,18 @@
 packages:=boost openssl zeromq libiconv
 
+# ccache is useless in gitian builds
+ifneq ($(GITIAN),1)
 native_packages := native_ccache
+endif
 
 hardware_packages := hidapi protobuf libusb
 hardware_native_packages := native_protobuf
 
 android_native_packages = android_ndk
 android_packages = ncurses readline sodium
 
-darwin_native_packages = native_biplist native_ds_store native_mac_alias $(hardware_native_packages)
-darwin_packages = sodium ncurses readline $(hardware_packages)
+darwin_native_packages = $(hardware_native_packages)
+darwin_packages = ncurses readline sodium $(hardware_packages)
```

Comment:
Just out of curiosity: does the difference in ordering change something here?

Reply:
No difference, I was just lining this up to be uniform with the other platforms.